_data/projects/hyperviz.yml

```diff
@@ -0,0 +1,11 @@
+name: HyperViz
+desc: Analysing hyper-parameters made easy.
+site: https://github.com/F1nnM/HyperViz
+tags:
+  - python
+  - streamlit
+  - web
+  - machine-learning
+upforgrabs:
+  name: hacktoberfest
```

Comment:
@F1nnM hacktoberfest will go away, want to rename it to something general such as `good first issue`? We can keep it to hacktoberfest if you want.

Reply:
Ah yes. Changed it.